m4_sessions/1711-inning/Group_8.md

```diff
@@ -15,19 +15,39 @@ Session Outcomes: What will students leave with or gain from this session?
     Support other students who are currently in a fixed-mindset. 
   
 
-Session Outline: Include the flow of the session and the time for each section.
-  - Introductions/Why we decided to provide this Gearup.
-  - Provide resources on how to identify a fixed mindset. 
-  - Students will identify an area that they are in a fixed mindset.
-  - Discuss the power of your self narrative. 
-  - Break out into groups to identify techniques to shift their narrative.
-  - Students will then breakout individually to rewrite their narrative. 
+# Session Outline: Include the flow of the session and the time for each section.
 
-Names of group members: Samuel Nayrouz, Francy Lang, Rufus Walsh, Ricky Amparo
+ ALL 10 minutes - Introductions/Why we decided to provide this Gearup. Share why we think we need this GearUp, what in our own personal journey has caused us to experience. 
```

Comment:
This could go long, so modeling what you want people to share will be important.